tests/Namshi/JOSE/Test/JWSTest.php

```diff
@@ -96,6 +111,6 @@ public function testLoadingAMalformedTokenString()
      */
     public function testLoadingAMalformedTokenString2()
     {
-        JWS::load('test', true);
```

Comment:
these boolean args were useless right? I dont think we have a bool argument there at all :)